ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -8,16 +8,34 @@ import {
   ButtonVariant,
 } from '../../../../../components/component-library';
 import { Footer as PageFooter } from '../../../../../components/multichain/pages/page';
-import { currentConfirmationSelector } from '../../../../../selectors';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import { useMMIConfirmationInfo } from '../../../../../hooks/useMMIConfirmations';
+import { doesAddressRequireLedgerHidConnection } from '../../../../../selectors';
 import {
   rejectPendingApproval,
   resolvePendingApproval,
 } from '../../../../../store/actions';
-import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import { currentConfirmationSelector } from '../../../selectors';
 
 const Footer = () => {
   const t = useI18nContext();
   const currentConfirmation = useSelector(currentConfirmationSelector);
+  ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
+  const { mmiOnSignCallback, mmiSubmitDisabled } = useMMIConfirmationInfo();
+  ///: END:ONLY_INCLUDE_IF
+
+  let from: string | undefined;
+  // todo: extend to other confirmation types
+  if (currentConfirmation?.msgParams) {
+    from = currentConfirmation?.msgParams?.from;
+  }
```

Comment:
```suggestion
const from: string | undefined = currentConfirmation?.msgParams?.from;
```

Reply:
Hey @pedronfigueiredo : I wrote it like this as other transaction type like send, approve etc will have different logic to initialise `from`.